### DISCUSSION TITLE
The implementation of SUPIR in Forge (Scaling Up to Excellence: Practicing Model Scaling for Photo-Realistic Image Restoration)

### DISCUSSION THREAD

**Discussion by ivaxsirc** [lllyasviel/stable-diffusion-webui-forge#431] [Ideas]
SUPIR https://github.com/Fanghua-Yu/SUPIR is the best upscaler of the moment Dr. Furkan has tested and the results are awesome,

https://www.youtube.com/watch?v=PqREA6-bC3w&feature=youtu.be

He has managed to make it work with runs with 12GB VRAM, but with all the features needs 32GB of VRAM.
I'm sure that forge could be an implimentation to need less VRAM

Thanks

**Comment by davizca** [comment]
Will be awesome seeing that in Forge.

**Reply by scarbain** [reply]
I created a feature request in the issues! 
https://github.com/lllyasviel/stable-diffusion-webui-forge/issues/451

**Reply by ivaxsirc** [reply]
OMG look this new module in SUPIR for the faces, awesome!!
![309478841-6b91a69e-c20f-4e2a-bfc8-6dcb4fb1173f](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/48472866/7e932b24-1382-4377-ab49-a6c3f9e075a1)
![309478910-3b77b533-a445-4bb7-b1a7-3735c7281f75](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/48472866/2d3ae585-e6f6-4de2-a280-21e2af466932)

**Comment by ivaxsirc** [comment]
more images with supir 
original
![Imagen de WhatsApp 2024-03-03 a las 18 55 49_acc05a71](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/48472866/1169804d-f54e-47bd-9e5b-c43de24fdd7a)
![image (35)](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/48472866/c64c707d-d777-44e2-90d1-772509207b2d)

**Comment by alcoartist** [comment]
I'll be glad if this happens.
Just don't mention this shady guy from patreon (furkan money boy). He is NOT the developer of SUPIR, just milking it for his own pocket.
